compiler/AST/iterator.cpp

```diff
@@ -1159,6 +1232,8 @@ void lowerIterator(FnSymbol* fn) {
 
   BlockStmt* singleLoop = NULL;
   if (CallExpr* singleLoopYield = isSingleLoopIterator(fn, asts))
+    // If the iterator contains a single loop statement containing a single
```

Comment:
The Microsoft DevDiv style guide says every conditional block requires braces (on separate lines, no less), a style that I favor increasingly.  For one thing, it makes each conditional occupy at least 4 lines of code, which adds teeth to my mantra, "Every 'if' costs you money."
Braces added.